src/status_im/contexts/chat/messenger/messages/content/lightbox/utils.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.contexts.chat.messenger.messages.content.lightbox.utils
+  (:require
+    [utils.datetime :as datetime]
+    [utils.re-frame :as rf]))
+
+(defn convert-message-to-lightbox-image
+  [{:keys [timestamp image-width image-height message-id from content]}]
+  (let [[primary-name _] (rf/sub [:contacts/contact-two-names-by-identity from])]
```

Comment:
Not sure an utils function should susbcribe, I think they should be more pure, maybe it can be an additional parameter?

Reply:
To be honest, I see no convenient way to remove this subscription out of the function. It is used within the map in `status-im.contexts.chat.messenger.messages.content.album.view` and subscription itself relies on other subs, so can't be easily converted to usual function. I would prefer to leave it "as is" because it encapsulates all the work needed to convert message to image description.